src/components/forms/language-panel/language-panel.styles.js

```diff
@@ -0,0 +1,22 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export const useStyles = makeStyles((theme) => ({
+  title: {
+    color: '#3f51b5',
+    padding: '5px'
+  },
+  inputPanel: {
+    display: 'flex',
+    flexDirection: 'column',
+    margin: '10px 0 !important',
```

Comment:
I don't like this `!important` statement 😄 Can you please dive into it to avoid it?

Reply:
fixed